pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/converters/BrevbakerBrevdataModule.kt

```diff
@@ -5,17 +5,31 @@ import com.fasterxml.jackson.databind.DeserializationContext
 import com.fasterxml.jackson.databind.JsonDeserializer
 import com.fasterxml.jackson.databind.module.SimpleModule
 import no.nav.pensjon.brev.api.model.maler.BrevbakerBrevdata
+import no.nav.pensjon.brev.api.model.maler.RedigerbarBrevdata
 
 object BrevbakerBrevdataModule : SimpleModule() {
     private fun readResolve(): Any = BrevbakerBrevdataModule
 
     private class GenericBrevdata : LinkedHashMap<String, Any>(), BrevbakerBrevdata
 
+    private class GenericRedigerbarBrevdata : LinkedHashMap<String, Any>(), RedigerbarBrevdata<BrevbakerBrevdata, BrevbakerBrevdata> {
```

Comment:
Hmm... jeg lurer litt på om denne bare kan være `private data class GenericRedigerbarBrevdata(val saksbehandlerValg: GenericBrevdata, val pesysData: GenericBrevdata): RedigerbarBrevdata`

Reply:
Men det er også mulig at det vil skape problemer for `TemplateResource::parseArgument`.

Reply:
Det vart jo mykje enklare og betre, ja.